packages/flutter_driver/test/src/real_tests/stubs/stub_command_extension.dart

```diff
@@ -23,9 +23,8 @@ class StubNestedCommandExtension extends CommandExtension {
   }
 
   @override
-  Command deserialize(Map<String, String> params, DeserializeFinderFactory finderFactory, DeserializeCommandFactory commandFactory) {
-    return StubNestedCommand.deserialize(params, finderFactory);
-  }
+  Command deserialize(Map<String, String> params, DeserializeFinderFactory finderFactory, DeserializeCommandFactory commandFactory) =>
+    StubNestedCommand.deserialize(params, finderFactory);
```

Comment:
This looks quite a bit nicer!

Unfortunately, the Flutter repo has a [style guideline](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods) specifically for this situation:

> only use `=>` when everything, including the function declaration, fits on a single line.

Reply:
> This looks quite a bit nicer!
> 
> Unfortunately, the Flutter repo has a [style guideline](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods) specifically for this situation:
> 
> > only use `=>` when everything, including the function declaration, fits on a single line.

thank you for the info.